### DISCUSSION TITLE
Sequences bug?

### DISCUSSION THREAD

**Discussion by Lifeisgood7272** [onlaj/Piano-LED-Visualizer#266] [General]
Hey,

I got the sequences working. At first it crashed whenever I got to a step that had a light speed of "Very Fast", I figured it was because I spelled it wrong, so I tried "Very_Fast" and it still crashed. I switched it to "Instant" and now it works. So my first question is, how do I spell "Very Fast" light speed on the sequences.xml file so the Pi can read it? I'll try "VeryFast" next and see if that counts. Same for Very Slow.

**Also, the program appears to be skipping every other sequence, even on the examples you gave.** This is my biggest problem. Is this a known bug or is something wrong with the way it's written? Right now I can work around it by having dummy sequences every other sequence so it plays only the ones I want, but right now I have a sequence of 29 steps so that will be 58 steps because of this issue! I attached the file as a .txt file, let me know if anything looks wrong. (.xml not supported on github)
sequences - Copy.txt

**Comment by onlaj** [comment]
> how do I spell "Very Fast" light speed on the sequences.xml

"Very fast"
I'm sorry it wasn't clear and specified anywhere. I always had in mind making some tool for creating sequences in GUI. I'm currently working on it and it should be finished before the end of this week.

> Also, the program appears to be skipping every other sequence, even on the examples you gave.

I have never run into this problem. Does it happen both when you choose the activation method for the pedal press and the button? What about "next step" button on the web interface, still the same problem?
